File: 백준/Bronze/2577. 숫자의 개수/숫자의 개수.py
a = int(input())
b = int(input())
c = int(input())
num = a * b * c
num = str(num)

arr = [0 for _ in range(10)]

for item in num:
  arr[int(item)] += 1

for item in arr:
  print(item, end = ' ')

File: 백준/Bronze/2577. 숫자의 개수/숫자의 개수.c
#include <stdio.h>

int main(void) {  
   int a, b, c;
   scanf("%d", &a);
   scanf("%d", &b);
   scanf("%d", &c);

   int arr[10] = {0};
   int num = a * b * c;
   while(1){
      if (num == 0){
         break;
      }
      int k = num % 10;
      for(int i = 0; i < 10; i++){
         if (k == i){
            arr[i] += 1;
         }
      }

      num -= k;
      num /= 10;
   }

   for(int i = 0; i < 10; i++){
      printf("%d ", arr[i]);
   }

   return 0;
}